src/styles/_catalog.scss

```
@@ -1,9 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 487px) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
```

```suggestion
      grid-template-columns: repeat(1, 200px);
```
Нехай це буде дефолтне значення а при збільшені екрану перезаписується